packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -269,3 +269,45 @@ export const assertUsesNatMath = (zcf, brand) => {
     details`issuer must use NAT amountMath`,
   );
 };
+
```

Comment:
Hmm, this doesn't seem to belong in this PR

Reply:
it's because I built from non-master. I just rebased the PR so the difference is correct.